python/ccf/proposal_generator.py

```diff
@@ -231,11 +237,6 @@ def new_member(
     return false
     end
 
-    expected_enc_pub_key = [====[{encryption_pub_key}]====]
```

Comment:
This is now optional so the sample vote should only check for the member's identity public key.

Reply:
I think this should remain, wrapped in `if encryption_pub_key is not None:`. If the proposal _does_ specify an encryption public key, then the careful vote for it should check that encryption public key is present.

Reply:
Sorry this is inline Lua. So either formatted or hand-appended or something, if we have an encryption key in the Python we should check it in the Lua.

Reply:
You're absolutely right. I'd assumed that checking for the member's identity certificate was sufficient for some reason anyway. Fixed now.